### ISSUE TITLE
Plugin discovered as [-packer-default-plugin-name-] instead of [windows-update]

### ISSUE THREAD

**Issue by frippe75** [rgl/packer-plugin-windows-update#92]
I placed it in the /bin directory which maybe was a mistake?

But the plugin is not being discovered correctly and later it fails on a "unknown provisioner".

Ideas??

```
2021/06/14 07:58:24 [INFO] Packer version: 1.7.2 [go1.16.3 linux amd64]
2021/06/14 07:58:24 [TRACE] discovering plugins in /bin
2021/06/14 07:58:24 [DEBUG] Discovered plugin: windows-update_v0.12.0_x5.0_linux_amd64 = /bin/packer-plugin-windows-update_v0.12.0_x5.0_linux_amd64
2021/06/14 07:58:24 found external [-packer-default-plugin-name-] provisioner from windows-update_v0.12.0_x5.0_linux_amd64 plugin
2021/06/14 07:58:24 [TRACE] discovering plugins in /root/.packer.d/plugins
2021/06/14 07:58:24 [TRACE] discovering plugins in .
2021/06/14 07:58:24 [INFO] PACKER_CONFIG env var not set; checking the default config file path
2021/06/14 07:58:24 [INFO] PACKER_CONFIG env var set; attempting to open config file: /root/.packerconfig
2021/06/14 07:58:24 [WARN] Config file doesn't exist: /root/.packerconfig
2021/06/14 07:58:24 [INFO] Setting cache directory: /image/packer_cache
2021/06/14 07:58:24 [TRACE] Starting internal plugin packer-builder-vsphere-iso

==> Builds finished but no artifacts were created.
2021/06/14 07:58:24 packer-provisioner-powershell plugin: Serving a plugin connection...

error initializing provisioner 'windows-update': Unknown provisioner


```

**Comment by frippe75** [comment]
Tried placing it in the .packer.d/plugins directory with the same result

```
2021/06/14 08:09:06 [INFO] Packer version: 1.7.2 [go1.16.3 linux amd64]
2021/06/14 08:09:06 [TRACE] discovering plugins in /bin
2021/06/14 08:09:06 [TRACE] discovering plugins in /root/.packer.d/plugins
2021/06/14 08:09:06 [DEBUG] Discovered plugin: windows-update_v0.12.0_x5.0_linux_amd64 = /root/.packer.d/plugins/packer-plugin-windows-update_v0.12.0_x5.0_linux_amd64
2021/06/14 08:09:06 found external [-packer-default-plugin-name-] provisioner from windows-update_v0.12.0_x5.0_linux_amd64 plugin
```

**Comment by frippe75** [comment]
I guess I found the answer to this. I installed it the old-fashion-way since I'm in a disconnect env.
Wasn't aware of the details. Had to place it in the plugins dir and rename it from packer-plugin-windows-update_v0.12.0_x5.0_linux_amd64 to packer-plugin-windows-update exactly.

The plugin name will be what ever comes after packer-plugin-*****   and the platform part (amd64...) got in the way.

Now lets see if my image builds...
